instance/tests/models/test_load_balancer.py

```diff
@@ -47,7 +47,7 @@ def mock_instances():
     """
     instances = [
         _make_mock_instance(
-            ["test1.lb.opencraft.hosting", "test2.lb.opencraft.hosting"],
+            ["test1.lb.opencraft.hosting", "TEST2.lb.opencraft.hosting"],
```

Comment:
@smarnach Here I see why it's useful to put the test in a shared method, since it will try it on all the tests, which is better than trying it in one. To remember that this caused a bug at one point, it could be worth adding a comment, so the test isn't removed inadvertently.